services/mgmt/containerservice/README.md

```diff
@@ -6,70 +6,12 @@ https://github.com/Azure/azure-rest-api-specs/blob/main/specification/containers
 
 To get started with these generated service crates, see the [examples](https://github.com/Azure/azure-sdk-for-rust/blob/main/services/README.md#examples).
 
-The default tag is `package-2022-03`.
```

Comment:
There will probably be a few places where sorting alphabetically doesn't work. This is one. I don't think we need to fix every case in this PR though.